### ISSUE TITLE
NaN's introduced during training.

### ISSUE THREAD

**Issue by charliebudd** [facebookresearch/barlowtwins#53]
I'm seeing NaN's being introduced whilst training with the default configuration on ImageNet. The loss spiked suddenly, after which the loss was returned as NaN. This has occurred twice in a row. Similar issues exist in other facebookresearch repos https://github.com/facebookresearch/vissl/issues/543, the common link perhaps being the near identical implementation of the LARS optimiser. The linked issue suggests setting the optimiser to exclude bias and norm to avoid NaNs, however this is different to the published BarlowTwins method. 

**EDIT:** The BarlowTwins paper does in fact state that the bias and batch normalization parameters are excluded from the LARS optimiser and this is the case in the code. Perhaps given the spike, this is an exploding gradients problem?

### Loss graph:
![image](https://user-images.githubusercontent.com/72921588/201892488-9afc1209-ab01-484d-8a1c-c7dbaa3b921f.png)
